lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgstr "N/A"
+
+# MSG_NO c=4
+#: messages.c:71
+msgid "No"
+msgstr "Nej"
+
+# MSG_SELFTEST_NOTCONNECTED c=20
+#: ultralcd.cpp:8128
+msgid "Not connected"
+msgstr "Inte ansluten"
+
+# MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2
+#: util.cpp:195
+msgid "New firmware version available:"
+msgstr "Ny firmware version tillg\xe1nglig:"
+
+# MSG_SELFTEST_FAN_NO c=19
+#: messages.c:92
+msgid "Not spinning"
+msgstr "Roterar inte"
+
+# MSG_WIZARD_V2_CAL c=20 r=8
+#: ultralcd.cpp:4838
+msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
+msgstr "Nu ska jag kalibrera avstandet mellan munstyckets spets och v\xe1rmeb\xe1ddsytan."
+
+# MSG_WIZARD_WILL_PREHEAT c=20 r=4
+#: ultralcd.cpp:4967
+msgid "Now I will preheat nozzle for PLA."
+msgstr "Nu ska jag f\xefrv\xe1rma munstycket f\xefr PLA."
+
+# MSG_NOZZLE c=12
+#: messages.c:72
+msgid "Nozzle"
+msgstr "Munstycke"
+
+# MSG_DEFAULT_SETTINGS_LOADED c=20 r=6
+#: Marlin_main.cpp:1605
+msgid "Old settings found. Default PID, Esteps etc. will be set."
+msgstr "Gamla inst\xe1llningar hittades. Standard PID, Esteps etc. kommer att st\xe1llas in."
+
+# MSG_REMOVE_TEST_PRINT c=20 r=4
+#: ultralcd.cpp:4958
+msgid "Now remove the test print from steel sheet."
+msgstr "Ta nu bort testutskriften fran metallskivan."
+
+# MSG_NOZZLE_FAN c=10
+#: ultralcd.cpp:1446
+msgid "Nozzle FAN"
+msgstr "Muns fl\xe1kt"
+
+# MSG_PAUSE_PRINT c=18
+#: messages.c:74
+msgid "Pause print"
+msgstr "Pausa utskrift"
+
+# MSG_PID_RUNNING c=20
+#: ultralcd.cpp:1327
+msgid "PID cal."
+msgstr "PID kalibrering."
+
+# MSG_PID_FINISHED c=20
+#: ultralcd.cpp:1333
+msgid "PID cal. finished"
+msgstr "PID kalibrering klar"
+
+# MSG_PID_EXTRUDER c=17
+#: ultralcd.cpp:5799
+msgid "PID calibration"
+msgstr "PID kalibrering"
+
+# MSG_PINDA_PREHEAT c=20
+#: ultralcd.cpp:683
+msgid "PINDA Heating"
+msgstr "PINDA uppv\xe1rmning"
+
+# MSG_PAPER c=20 r=10
+#: messages.c:73
+msgid "Place a sheet of paper under the nozzle during the calibration of first 4 points. If the nozzle catches the paper, power off the printer immediately."
+msgstr "Placera ett pappersark under munstycket under kalibreringen av de f\xefrsta 4 punkterna. St\xe1ng av skrivaren omedelbart om munstycket r\xefr vid pappret.."
+
+# MSG_WIZARD_CLEAN_HEATBED c=20 r=8
+#: ultralcd.cpp:5024
+msgid "Please clean heatbed and then press the knob."
+msgstr "Reng\xefr b\xe1dden och tryck sedan pa knappen."
+
+# MSG_CONFIRM_NOZZLE_CLEAN c=20 r=8
+#: messages.c:26
+msgid "Please clean the nozzle for calibration. Click when done."
+msgstr "Reng\xefr munstycket f\xefr kalibrering. Klicka n\xe1r du \xe1r klar."
+
+# MSG_SELFTEST_PLEASECHECK c=20
+#: ultralcd.cpp:8122
+msgid "Please check:"
+msgstr "Kontrollera:"
+
+# MSG_WIZARD_CALIBRATION_FAILED c=20 r=8
+#: messages.c:117
+msgid "Please check our handbook and fix the problem. Then resume the Wizard by rebooting the printer."
+msgstr "Se var handbok och atg\xe1rda problemet. Ateruppta sedan guiden genom att starta om skrivaren."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.